View/encoding_table.py

```diff
@@ -173,6 +176,36 @@ def get_col_count(self):
         """
         return self.columnCount()
 
+    def get_cell_size(self):
+        """
+        Getter method to get cell size.
+
+        Returns:
+            list
```

Comment:
Add more description to the return type. List of size what, and containing what data?

Note: Since this data is of a fixed-size, and we're re-assigning the variable with updates (and not directly manipulating the data inside), it might be worth switching this type to a tuple. Syntax: (min_width, min_height). Just an idea.